cmd/oras/internal/option/remote.go

```diff
@@ -113,10 +116,16 @@ func (opts *Remote) Parse() error {
 	return opts.distributionSpec.Parse()
 }
 
-// readPassword tries to read password with optional cmd prompt.
+// readPassword tries to read password and identity-token with optional cmd prompt.
 func (opts *Remote) readPassword() (err error) {
```

Comment:
I'm fine with this, but there is a Cobra feature for mutually exclusive

Reply:
Oh, thanks for the info. I will check it and get back on this one.

Reply:
Please refer to https://pkg.go.dev/github.com/spf13/cobra#Command.MarkFlagsMutuallyExclusive

Reply:
BTW `identity-token` is exclusive to both `password` and `username`